code-studio/src/js/ajaxSubmit.js

```diff
@@ -6,7 +6,7 @@ module.exports = function ajaxSubmit(form_selector) {
       xhr.setRequestHeader('X-CSRF-TOKEN', token);
     });
     $(form_selector).on('ajax:complete', function (e, data) {
-      if (data.status == "200") {
+      if (data.status === 200) {
```

Comment:
This one is perhaps slightly concerning in that in the instance I looked at data.status was a number, and thus we were depending on double equals. I fixed to use triple equals and compare to a number, but this will break if data.status could also be a string.

I believe this is just ajax and should always be a number.